core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -98,8 +101,8 @@
    * stream reads from a remote worker.
    */
   protected BlockOutStream mCurrentCacheStream;
-  /** The blockId used in the block streams. */
-  private long mStreamBlockId;
+  /** Index of the ID of the current block stream in {@link #mBlockIds}. */
```

Comment:
can you doc here: what does a value`-1` indicates?

Reply:
add doc